boot/LoadLinux.c

```diff
@@ -306,13 +306,18 @@ CONFIGENTRY *LoadConfigCD(int cdromId) {
 	int n;
 	int configLoaded=0;
 	CONFIGENTRY *config, *currentConfigItem;
+	int nTempCursorX, nTempCursorY;
 
 	memset((u8 *)KERNEL_SETUP,0,4096);
 
 	printk("\2Please wait\n\n");
 	//See if we already have a CDROM in the drive
 	//Try for 8 seconds - takes a while to 'spin up'.
+	nTempCursorX = VIDEO_CURSOR_POSX;
+	nTempCursorY = VIDEO_CURSOR_POSY;
+again:
 	I2CTransmitWord(0x10, 0x0c01); // close DVD tray
+	printk("Loading linuxboot.cfg from CDROM... \n");
```

Comment:
nit: "CD-ROM"

Reply:
"CDROM" "CD-ROM" both are used interchangeably in many different applications.

Reply:
I decided to use just "CD" here, because it's simpler and shorter. Also fixed in other places for consistency.